src/renderer/components/prompts/folder.jsx

```diff
@@ -23,25 +22,21 @@ export default React.createClass({
 
   render: function () {
 
-    const raisedButtonStyle = {
-      margin: '56px 24px 0px 24px'
-    };
-
     const isEmpty = !!this.props.selectedFolder;
 
     return (
-      <div className="select-folder-prompt fieldset">
-        <label style={{ background: this.props.color }}>
+      <div style={this.getContainerStyle()}>
+        <label style={this.getLabelStyle()}>
```

Comment:
The [React Native Web](https://github.com/necolas/react-native-web) project has some great [examples](https://github.com/necolas/react-native-web/blob/master/docs/components/Text.md#examples) of building abstractions into components for styling.